packages/core/frontend/sirius-components-core/src/workbench/Workbench.types.ts

```diff
@@ -42,28 +43,68 @@ export interface WorkbenchViewContribution {
 export interface WorkbenchViewComponentProps {
   editingContextId: string;
   readOnly: boolean;
+  initialConfiguration: WorkbenchViewConfiguration | null;
 }
 
 export interface MainAreaComponentProps {
   editingContextId: string;
   readOnly: boolean;
+  initialConfiguration: WorkbenchMainPanelConfiguration | null;
```

Comment:
Why this change, is this used? The main area is the onboard area in Sirius Web, it doesn't have any editor at all.